app/views/katello/api/v2/sync_plans/show.json.rabl

```diff
@@ -31,3 +31,7 @@ end
 node :sync_date do |sync_plan|
   sync_plan.plan_date_time
 end
+
+node :enabled do |sync_plan|
+  sync_plan.foreman_tasks_recurring_logic.state == 'active'
+end
```

Comment:
@jlsherrill : For the enabled flag on sync plan, we get the flag from rec logic directly..The value that API returns and what's stored on the field might be different..Do we want a task to correct that?